client/src/lib/Channel.ts

```diff
@@ -41,6 +43,16 @@ export class Channel {
     }
     return encryptCEKForVerificationMethod(this.cek, verificationMethod);
   }
+  
+  async hasMember(did: PublicKey):Promise<boolean> {
```

Comment:
I assume this is not directly related to the dev env.

Reply:
not directly. I was debugging adding accounts to the new public channel on the dev env. I noticed if you clear your browser cache then re-add the same DID, it would throw an error, so this function is to help with that.